src/pages/docs/developer-guide/index.js

```diff
@@ -57,14 +57,8 @@ const Doc = class ReduxDocList extends Component {
 export const devGuideQuery = graphql`
 	query devGuideQuery {
 		guidesList: allMarkdownRemark(
-			filter:{
-				fields:{
-					slug: {regex: "/docs\\/developer-guide\\/[^/]*\/$/"}
-				}
-			},
-			sort: {
-				fields: [frontmatter___order, frontmatter___title], order: ASC
-			}
+			filter: {fields: {slug: {regex: "/docs\\/developer-guide\\/[^/]*/$/"}}},
```

Comment:
Could you please reformat to the version on develop? It would be better for readability. Please do for every query